<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="CtaContOperSecundaria.contrato_operSecundarias-failed" tests="1" file="CtaContOperSecundaria.py" time="71.533" timestamp="2024-10-09T15:36:55" failures="0" errors="1" skipped="0">
	<testcase classname="CtaContOperSecundaria.contrato_operSecundarias" name="test_contratc_oparSecondary" time="71.533" timestamp="2024-10-09T15:36:55" file="CtaContOperSecundaria.py" line="22">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=129.0.6668.90)
Stacktrace:
	GetHandleVerifier [0x00007FF642E1B675+29573]
	(No symbol) [0x00007FF642D904A0]
	(No symbol) [0x00007FF642C4B6EA]
	(No symbol) [0x00007FF642C1FCD5]
	(No symbol) [0x00007FF642CCEF67]
	(No symbol) [0x00007FF642CE7FC1]
	(No symbol) [0x00007FF642CC70A3]
	(No symbol) [0x00007FF642C912DF]
	(No symbol) [0x00007FF642C92441]
	GetHandleVerifier [0x00007FF64314C5BD+3375821]
	GetHandleVerifier [0x00007FF6431979B7+3684039]
	GetHandleVerifier [0x00007FF64318CDDB+3640043]
	GetHandleVerifier [0x00007FF642EDB7F6+816390]
	(No symbol) [0x00007FF642D9B7AF]
	(No symbol) [0x00007FF642D975D4]
	(No symbol) [0x00007FF642D97770]
	(No symbol) [0x00007FF642D865CF]
	BaseThreadInitThunk [0x00007FFFD2F3257D+29]
	RtlUserThreadStart [0x00007FFFD420AF08+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\CtaContOperSecundaria.py", line 92, in test_contratc_oparSecondary
    find_elements(self.driver, selet_list_contract )
  File "c:\Users\luist\repos\Automatizaciones_M-dulos-\Elements.py", line 19, in find_elements
    select_nex_button = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 363, in _predicate
    target = driver.find_element(*target)  # grab element at locator
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 748, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 354, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=129.0.6668.90)
Stacktrace:
	GetHandleVerifier [0x00007FF642E1B675+29573]
	(No symbol) [0x00007FF642D904A0]
	(No symbol) [0x00007FF642C4B6EA]
	(No symbol) [0x00007FF642C1FCD5]
	(No symbol) [0x00007FF642CCEF67]
	(No symbol) [0x00007FF642CE7FC1]
	(No symbol) [0x00007FF642CC70A3]
	(No symbol) [0x00007FF642C912DF]
	(No symbol) [0x00007FF642C92441]
	GetHandleVerifier [0x00007FF64314C5BD+3375821]
	GetHandleVerifier [0x00007FF6431979B7+3684039]
	GetHandleVerifier [0x00007FF64318CDDB+3640043]
	GetHandleVerifier [0x00007FF642EDB7F6+816390]
	(No symbol) [0x00007FF642D9B7AF]
	(No symbol) [0x00007FF642D975D4]
	(No symbol) [0x00007FF642D97770]
	(No symbol) [0x00007FF642D865CF]
	BaseThreadInitThunk [0x00007FFFD2F3257D+29]
	RtlUserThreadStart [0x00007FFFD420AF08+40]

]]></error>
		<system-out><![CDATA[¡Elemento encontrado y clickeado 2 veces con éxito!
¡Elemento encontrado y clickeado 9 veces con éxito!
Tiempo de espera agotado. El texto por texto no está presente.
]]></system-out>
	</testcase>
</testsuite>
